server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -2360,7 +2360,7 @@ public void run() {
         zoo.putPersistentData(zPath, new byte[0], NodeExistsPolicy.SKIP);
 
         if (tabletServerLock.tryLock(lw, lockContent)) {
-          log.debug("Obtained tablet server lock " + tabletServerLock.getLockPath());
+          log.info("Obtained tablet server lock " + tabletServerLock.getLockPath());
```

Comment:
I am not sure, this was on the code before I worked on it, I can check it out though.